src/regressor.jl

```diff
@@ -47,6 +50,7 @@ A private method that returns the shape of the input and output of the model for
 data `X` and `y`.
 """
 shape(model::MultitargetNeuralNetworkRegressor, X, y) = (ncols(X), ncols(y))
+is_embedding_enabled_type(::MultitargetNeuralNetworkRegressor) = true
 
```

Comment:
As you are now acting on instances instead of types, I'd change the name of your trait from `is_embedding_enabled_type` to `is_embedding_enabled`, but this is just a suggestion.

Reply:
Sure thing.